docs/source/model_design/general_research.rst

```diff
@@ -679,11 +679,131 @@ Conducting Systematic Literature Searches/Reviews
+
+.. image:: collider_dag.svg
+
+**In the case that two variables are linked through a pathway with a collider (and there is no directional or backdoor pathway between them), there will be no association or causal realtionship between the two variables.** This is also referred to as a **"closed path."**
+
+Summary
+~~~~~~~
+
+.. list-table:: DAG Pathways
+   :widths: 5 5 5 5 5
+   :header-rows: 1
+
+   * - Path Name/Term
+     - Description
+     - Direction
+     - Open or closed
+     - Association
+   * - Direct path
+     - Path with arrows flowing in same direction
+     - Directed
+     - Open
+     - Causal relationship
+   * - Open backdoor path
+     - Path with arrows flowing in different directions without a collider
+     - Undirected
+     - Open
+     - Non-causal association
+   * - Closed path
+     - Path with arrows flowing in different directions with a collider
+     - Undirected
+     - Closed
+     - No association
+
+Identifying Confounding Using Directed Acyclic Graphs
++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+As discussed previously in this documentation, confounding arises when a given exposure and outcome share a common cause (Confounding_). This section will help to identify potential confounding variables in a given causal network that may be less explicit than the example(s) presented in the Confounding_ section.
+
+For the remainder of this section, we will focus on the relationship between a particular exposure (E), shown in blue, and a particular outcome (O), shown in purple.
+
+In the context of DAGs, confounding may occur when there is an *open* backdoor pathway (*unblocked* backdoor pathway with no colliders) between an exposure and outcome of interest. Therefore, any variable that can be used to block an open backdoor path between a given exposure and outcome is a *confounder* with regard to the relationship between that exposure and outcome. 
+
+"Blocking" an open backdoor path can be done via randomization of that variable between exposure groups in the context of an randomized controlled trial or via conditioning (adjustment via stratification or other methods) in the context of an observational study  on that variable. In the following diagrams, variables that are blocked in this way will be represented as squares rather than circles; when this is the case, assume that a pathway with a square variable is a *closed* pathway. Notably, it is not necessary to condition on all variables on a backdoor path; the path can be closed by conditioning on a minimal set of variables and there may be more than one possible minimal set of variables. See below for possible options for closing backdoor paths by conditioning a *minimal set* of variables (in this case a single variable).
```

Comment:
```suggestion
"Blocking" an open backdoor path can be done via randomization of that variable between exposure groups in the context of a randomized controlled trial or via conditioning (adjustment via stratification or other methods) in the context of an observational study  on that variable. In the following diagrams, variables that are blocked in this way will be represented as squares rather than circles; when this is the case, assume that a pathway with a square variable is a *closed* pathway. Notably, it is not necessary to condition on all variables on a backdoor path; the path can be closed by conditioning on a minimal set of variables and there may be more than one possible minimal set of variables. See below for possible options for closing backdoor paths by conditioning a *minimal set* of variables (in this case a single variable).
```